WeightProducer/python/EMJxsecs.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+from TreeMaker.WeightProducer.MCSampleValues import MCSampleValuesHelper
+XSValues = MCSampleValuesHelper.XSValues
+EMJxsecs = {
+"mMed-1000": {"CrossSection":XSValues(XS_13TeV=0.02049) },
```

Comment:
Please follow the example in [1] and put an `XSSource_13TeV` reference. This will help us keep track of where the values come from.

[1]
https://github.com/TreeMaker/TreeMaker/blob/9279e89713ed2999abe53d62a562ec363f8f645b/WeightProducer/python/MCSampleValues.py#L123

Reply:
Cross section sources has been added